.github/workflows/scan.yml

```diff
@@ -36,7 +36,7 @@ jobs:
         uses: anchore/scan-action@v6
         id: scan_container
         with:
-          image: "ghcr.io/freedomofpress/dangerzone/dangerzone:${{ steps.tag.outputs.tag }}"
```

Comment:
It seems that this is missing the update of the image name in `release-container-image.yml` workflow, in order to publish the `freedomofpress/dangerzone/v1` image nightly.

Reply:
Whoops, nice catch. Fixed in 61002cb3.